src/segment-loader.js

```diff
@@ -833,6 +833,7 @@ export default class SegmentLoader extends videojs.EventTarget {
     }
 
     if (!oldPlaylist || oldPlaylist.uri !== newPlaylist.uri) {
+      this.trigger('playlist-update');
```

Comment:
new event for detecting a playlist change from the loader alone. We need this because during playlist changes it can look like playback is stalled, when really it isn't.